app/src/main/res/layout/download_statistics_listitem_total_size.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This file can probably be shared, too. The only difference is the text of the header. That text can also be set in Java.